Controllers/PeopleController.cs

```diff
@@ -25,9 +28,15 @@ public IEnumerable<Person> Get()
         }
 
         [HttpPost]
-        public bool Post([FromBody] Person person)
+        public void Post([FromBody] Person person)
```

Comment:
This should probably return an IActionResult for the return type plus should then return a status code of either 200 for success or 204 for no content (meaning nothing to delete)